apps/condo/domains/organization/schema/OrganizationEmployeeRole.js

```diff
@@ -74,6 +74,11 @@ const OrganizationEmployeeRole = new GQLListSchema('OrganizationEmployeeRole', {
             type: Checkbox,
             defaultValue: true,
         },
+        isDivisionLimitedVisibility: {
```

Comment:
Maybe, this name will better reflect the logic behind this field?
Is this limitation only for reading operation?
```suggestion
        canReadEntitiesOnlyInScopeOfDivision: {
```

Will this limitation cover update, delete and create operations?
For this case, my suggestion is following:
```suggestion
        canOperateWithEntitiesOnlyInScopeOfDivision: {
```

Reply:
This limitation only for reading operation.
Agree that the name `canReadEntitiesOnlyInScopeOfDivision` is better